lib/Room.php

```diff
@@ -55,6 +55,10 @@ class Room {
 	public const TYPE_PUBLIC = 3;
 	public const TYPE_CHANGELOG = 4;
 
+	public const RECORDING_STOP = 0;
```

Comment:
"Stop" feels wrong, it's a verb, so should be more "stopped". But I would actually prefer:

```suggestion
	public const RECORDING_NONE = 0;
```